geoadmin/deploy.sh

```diff
@@ -88,11 +88,8 @@ fi
 echo "${YELLOW}NPM install...${RESET}"
 npm install
 
-echo "${YELLOW}Cleaning previous build...${RESET}"
-npm run clean
-
 echo "${YELLOW}Building with catalog ${CATALOG_URL}...${RESET}"
-HISTORY_MODE=hash CATALOG_URL=${CATALOG_URL} npm run build -- --public-url ./
+HISTORY_MODE=hash CATALOG_URL=${CATALOG_URL} npm run build -- --cardViewSort=desc --catalogUrl="$CATALOG_URL" --public-url --pathPrefix="/browser/"./
```

Comment:
```suggestion
npm run build -- --historyMode=hash --cardViewSort=desc --catalogUrl="${CATALOG_URL}" --pathPrefix="/browser/"./
```

The old HISTORY_MODE and CATALOG_URL env variables don't exist anymore, so I've added them as CLI parameters directly.

I'm not sure what `--public-url` was meant to do, I can't remember that this parameter existed?!

Reply:
Note: The new version of STAC Browser is stricter and doesn't show a search button for the STAC API anymore. This is due to the fact that the server is missing the required conformance class `https://api.stacspec.org/v1.0.0/item-search`

See: https://github.com/radiantearth/stac-api-spec/tree/release/v1.0.0/item-search

Reply:
With regards to the locales, you may want to enable the locales de-CH, it-CH and fr-CH for Switzerland, which were thankfully provided by your colleague @p1d1d1. You can only really do this by changing the config.js, which makes me wonder why you don't just update the config.js in general instead of using all the CLI parameters.

Reply:
Thanks for your input, we wanted to use CLI parameters to make updating our upstream branch easier in the future.